retrieval/targetmanager.go

```diff
@@ -165,6 +165,7 @@ func (tm *TargetManager) Run() {
 	})
 
 	tm.running = true
+	log.Info("Target manager started.")
```

Comment:
There's nothing blocking in the startup of the target manager. Let's just print this line than and remove the one above.